modules/deploy/pages/deployment-option/cloud/terraform-provider.adoc

```diff
@@ -0,0 +1,101 @@
+= Redpanda Terraform Provider
+:description: Learn how to use the Redpanda Terraform provider to create and manage resources in Redpanda Cloud.
+:page-cloud: true
+:page-beta: true
+
+The https://registry.terraform.io/providers/redpanda-data/redpanda/latest[Redpanda Terraform provider^] interfaces with the Cloud API, so you can define your Redpanda infrastructure in Terraform configuration files. You can use the Terraform provider to create and manage the following resources in Redpanda Cloud:
+
+* ACLs
+* Clusters 
+* Networks
+* Resource groups
+* Topics
+* Users
+
+== Prerequisites
+
+. Install Terraform following the https://learn.hashicorp.com/tutorials/terraform/install-cli[Terraform documentation^].
+. Clone the https://github.com/redpanda-data/terraform-provider-redpanda[`terraform-provider-redpanda` Github repository^]:
++
+[,bash]
+----
+git clone https://github.com/redpanda-data/terraform-provider-redpanda.git
+----
+
+== Limitations
+
+*?? TO REVIEWERS: Can you please confirm that the provider is supported on Serverless and Azure? And what functionality is supported in Cloud API but not provider, like Connectors?*
```

Comment:
Serverless was recently added, Azure should also be supported if the Cloud API supports it. I will leave the final call to @gene-redpanda since he worked on a PR to bring feature parity between Cloud API and the TF provider

Reply:
Serverless yes, Azure I did not see in the product doc. I'll see if we can get that in there too.